ajax/libs/wangEditor/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wangEditor",
+  "version": "2.1.23",
+  "filename": "js/wangEditor.min.js",
+  "description": "wangEditor - 基于javascript和css开发的 Web富文本编辑器 轻量简洁易用开源免费(wangEditor based on javascript and css that makes it lightweight, easy to create simple, free and opensource.)",
+  "homepage": "https://wangeditor.github.io/",
+  "author": "wangfupeng <wangfupeng1988@163.com>",
```

Comment:
I was thought we just need the author name and mail info. and the GitHub link I think it's not necessary., so I follow the upstream's `bower.json`, not `package.json`.

If I modify as the followings will it better?
```js
  "author": {
    "name": "wangfupeng1988",
    "email": "wangfupeng1988@163.com",
    "url": "https://github.com/wangfupeng1988"
  },
```